From 732f759d6e574edd9635045b9a4fd7c84eb8c7da Mon Sep 17 00:00:00 2001
From: Birger Schacht <birger@rantanplan.org>
Date: Sun, 25 Oct 2020 09:42:38 +0100
Subject: [PATCH] Add support for overriding variables before they are used

---
 config.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config.in b/config.in
index 08703bef..38b86153 100644
--- a/config.in
+++ b/config.in
@@ -20,6 +20,8 @@ set $term alacritty
 # on the original workspace that the command was run on.
 set $menu dmenu_path | dmenu | xargs swaymsg exec --
 
+include /etc/sway/config-vars.d/*
+
 ### Output configuration
 #
 # Default wallpaper (more resolutions are available in @datadir@/backgrounds/sway/)
-- 
2.28.0

